sentry-android-fragment/build.gradle.kts

```diff
@@ -0,0 +1,73 @@
+import io.gitlab.arturbosch.detekt.Detekt
+import io.gitlab.arturbosch.detekt.extensions.DetektExtension
+
+plugins {
+    id("com.android.library")
+    kotlin("android")
+    jacoco
+    id(Config.QualityPlugins.gradleVersions)
+    id(Config.QualityPlugins.detektPlugin)
+}
+
+android {
+    compileSdkVersion(Config.Android.compileSdkVersion)
+
+    defaultConfig {
+        targetSdkVersion(Config.Android.targetSdkVersion)
+        minSdkVersion(Config.Android.minSdkVersionFragment)
+
+        versionName = project.version.toString()
+        versionCode = project.properties[Config.Sentry.buildVersionCodeProp].toString().toInt()
+
+        // for AGP 4.1
+        buildConfigField("String", "VERSION_NAME", "\"$versionName\"")
+    }
+
+    buildTypes {
+        getByName("debug")
+        getByName("release")
```

Comment:
we probably need to have its own `proguard-rules.pro`, we don't want this class to be renamed, see eg the `sentry-android-timber` module

Reply:
Sure, added in 1b538250268295dda65d75de73d254e5e4381946